automated/linux/docker-prune/prune.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+. ../../lib/sh-test-lib
+. ./prune-lib.sh
+
+OUTPUT="$(pwd)/output"
+RESULT_FILE="${OUTPUT}/result.txt"
+export RESULT_FILE
+
+! check_root && error_msg "You need to be root to run this script."
+create_out_dir "${OUTPUT}"
```

Comment:
maybe there are other bits in the files that can be placed in the lib too... I'm just asking.

Reply:
I can certainly have another look to move over anything else